cmd/vic-machine/configure/configure.go

```diff
@@ -111,6 +113,12 @@ func (c *Configure) processParams() error {
 		return err
 	}
 
+	var err error
+	usingDHCP := ip.IsUnspecifiedIP(c.Data.ClientNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.PublicNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.ManagementNetwork.IP.IP)
```

Comment:
This line could be extracted into a function since it's being used here and in `create.go`.